dashboard/test/ui/features/artist.feature

```diff
@@ -20,6 +21,7 @@ Scenario: Winning the first level
   And I wait to see ".congrats"
   And element ".congrats" is visible
   And I press "continue-button"
+  Then I close the dialog
```

Comment:
Yes.  This was the video popup, which you would expect to appear if you start with a clean state.